datauri/datauri.py

```diff
@@ -8,9 +8,9 @@
 RE_DATA_URI = re.compile(
     'data:[{unreserved}{reserved}{percent}]+'
     .format(
-        unreserved="A-Za-z0-9-_.~",
-        reserved=":/?#\[\]@!$&'()*+,;=",  # only square brackets are escaped
-        percent='%'))
+        unreserved=r"A-Za-z0-9-_.~",
+        reserved=r":/?#\[\]@!$&'()*+,;=",  # only square brackets are escaped
```

Comment:
hmm. backslash behaviour changes here or what?

Reply:
Nothing changes in behavior. You're getting runtime warning in tests about invalid escape sequence that Python remains the same (IDK from where it goes, to be honest). Flake8 also does warning here.

Reply:
Yes, it is:
> If one is not specified, the media type of the data URI is assumed to be text/plain;charset=US-ASCII

Reply:
ah yeah now i see. should've been `...\\[\\]...` in the original i guess, but a raw string is cleaner here indeed.